### ISSUE TITLE
Smiley/Emojis not displaying correctly

### ISSUE THREAD

**Issue by jrie** [jitsi/jitsi-meet-electron#417]
## Description

Smileys/Emojis are not displaying correctly in chat using Debian Unstable or Fedora 13
---

## Current behavior
The emojis are only partly filled out partly and are not rendered correctly, also in chat window as well as in preview.

See this two images, firstly Fedora 32, the second Debian Unstable:

![fedora32](https://user-images.githubusercontent.com/5701785/87148651-15430300-c2af-11ea-8612-c998723d260a.png)

![debian_unstable](https://user-images.githubusercontent.com/5701785/87148676-1e33d480-c2af-11ea-9ed4-292da0f31a8e.png)

---

## Expected Behavior

Emojis are rendered correctly in preview and chat window

---

## Steps to reproduce

Open a Jitisi Meet session, open the chat window and click to insert a emoji

---

## Environment details
Debian Unstable and Fedora 32

---

**Comment by saghul** [comment]
Do you get the same problem when using Jitsi Meet in the browser?

**Comment by jrie** [comment]
Yes. It is the same result. I tested in Firefox and Opera, both stable versions.

Opera:
![jitsi_meet_opera](https://user-images.githubusercontent.com/5701785/87151342-f6933b00-c2b3-11ea-8473-bbe6f2cce84a.png)

Firefox:
![jitsi_meet_firefox](https://user-images.githubusercontent.com/5701785/87151346-f7c46800-c2b3-11ea-8179-5b48874a401b.png)

**Comment by saghul** [comment]
I suspect a font is missing, thanks for the report!
